src/docs/getting-started/best-practice.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Best Practice
+---
+
+import BrowserWindow from "@site/src/components/BrowserWindow/BrowserWindow";
+import DynamicImage from "@site/src/components/DynamicImage";
+
+## Connect Wallet
+
+The following best practice outlines recommendations for the user interface to create a uniform user experience across dApps in the ecosystem.
+
+Before being able to interact with a wallet through Beacon, the user first needs to perform an action to trigger the wallet connection.
+
+### Button Content
+
+Within the default Beacon configuration where multiple wallets are supported, the connection button should be labeled <b>"Connect Wallet"</b>.
+
+1. Button is clearly visible e.g. distinct border, background color or drop shadow
+2. Button is labeled with "Connect Wallet"
+
+<BrowserWindow url="https://mydapp.com">
+
+<DynamicImage src="/img/beacon-pairing-ui.png" alt="Beacon Pairing" />
+
+</BrowserWindow>
+
+When only supporting a certain wallet, the label could directly indetify the wallet e.g. "Connect AirGap Wallet"
```

Comment:
Remove this, because we don't officially support connecting to only a specific wallet?